README.md

```diff
@@ -69,19 +70,30 @@ To run this Docker image, you need a basic understanding of Docker, Docker-Compo
 
 ## Getting started
 
-1. Create a `game` sub-directory on your Docker-Node in your game-server-directory (Example: `/srv/palworld`). Give it full ownership with `chown -R 1000:1000 game/` or permissions with `chmod 777 game`.
+1. Create a `game` sub-directory on your Docker-Node in your game-server-directory (Example: `/srv/palworld`).
+   - This directory will be used to store the game server files, including configs and savegames.
 2. Set up Port-Forwarding or NAT for the ports in the Docker-Compose file.
-3. Pull the latest version of the image with `docker pull jammsen/palworld-dedicated-server:latest`.
+3. Pull the latest version of the image with:
+
+    ```shell
+    docker pull jammsen/palworld-dedicated-server:latest
+    ```
+
 4. Download the [docker-compose.yml](docker-compose.yml) and [default.env](default.env).
 5. Set up the `docker-compose.yml` and `default.env` to your liking.
-   1. Refer to the [Environment-Variables](#environment-variables) section for more information.
-6. Start the container via `docker-compose up -d && docker-compose logs -f`.
-   1. Watch the log, if no errors occur you can close the logs with ctrl+c.
-7. Now have fun and happy gaming!
+   - Refer to the [Environment-Variables](#environment-variables) section for more information.
+6. Start the container with:
+
+    ```shell
```

Comment:
Do it in-line, we dont need that copy paste function here, its a sacrifice for the "Gettings started" for beeing short and to the point.